optimizer-fix-controls.md

```diff
@@ -70,3 +70,10 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
 - Possible values: `[0, 2147483647]`
 - This variable sets the threshold for the optimizer's heuristic strategy to select access paths. If the estimated rows for an access path (such as `Index_A`) is much smaller than that of other access paths (default `1000` times), the optimizer skips the cost comparison and directly selects `Index_A`.
 - `0` means to disable this heuristic strategy.
+
+### [`52869`](https://github.com/pingcap/tidb/issues/52869) <span class="version-mark">New in v8.1.0</span>
+
+- Default value: `OFF`
+- Possible values: `ON`, `OFF`
+- As stated in "Note" section of [Explain Statements Using Index Merge](/explain-index-merge.md), if the optimizer can choose the single index scan method (other than full table scan) for a query plan, the optimizer will not automatically use index merge.
+- This limitation can be disabled by turning on this variable. Disabling this limitation enables the optimizer to choose index merge automatically for more quries, but may also make the optimizer miss better execution plans. So It's recommended to do sufficient testing for specific scenarios before disabling this limitation to make sure it won't cause performance regressions.
```

Comment:
```suggestion
- As stated in the "Note" of [Explain Statements Using Index Merge](/explain-index-merge.md#examples), if the optimizer can choose the single index scan method (other than full table scan) for a query plan, the optimizer will not automatically use index merge.
- You can remove this limitation by enabling this variable. Removing this limitation enables the optimizer to choose index merge automatically for more quries, but might make the optimizer miss better execution plans. Therefore, it is recommended to do sufficient testing for specific scenarios before remove this limitation to make sure it will not cause performance regressions.
```